gatekeeper-service/deploy/service.yaml

```diff
@@ -20,16 +20,18 @@ spec:
         - containerPort: 8080
         resources:
           requests:
-            memory: "32Mi"
-            cpu: "50m"
+            memory: "256Mi"
```

Comment:
Do we really need to request such a high amount? Is it not enough to just increase the limit?

Reply:
that change was unintentional and has been removed